python/setup.py.in

```diff
@@ -60,5 +60,7 @@ setup(
     author='Skupper Project',
     author_email='skupper@googlegroups.com',
     url='https://skupper.io/',
-    license='Apache Software License'
+    license='Apache Software License',
+
+    zip_safe=False,
```

Comment:
This one is not relevant when --single-version-... is in effect, looks like. So this can be removed from the PR

Reply:
Anyways, I don't want to have no zip, so let's keep it consistent and say no to zip once for all.